### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by directionless**
While running `misspell` on a different codebase. I happened to notice that some misspellings in the osquery code base. So, I fixed them

### CODE REVIEW

**Comment by FritzX6** [comment]
Don't forget about `frecuency`:
https://github.com/facebook/osquery/blob/8f7ea728d269b6a1096b39cb5b05573b72e7190e/specs/darwin/launchd.table#L16

File: include/osquery/distributed.h [hunk 1]
```diff
@@ -194,7 +194,7 @@ class DistributedPlugin : public Plugin {
    */
   virtual Status writeResults(const std::string& json) = 0;
 
-  /// Main entrypoint for distirbuted plugin requests
+  /// Main entrypoint for distribute plugin requests
```
Comment (muffins) [inline include/osquery/distributed.h]: I think this actually should be `distributed`

**Comment by muffins** [comment]
ok to test

File: include/osquery/distributed.h (hunk 1 above)
Comment (directionless) [reply include/osquery/distributed.h]: good catch

**Comment by fmanco** [comment]
@directionless could you rebase this on experimental and target the experimental branch on the PR? This won't merge cleanly as you're changing CMake files which were removed. Thank you.

**Pull request closed without merging**